k-distribution/k-tutorial/1_basic/14_evaluation_order/README.md

```diff
@@ -292,8 +292,7 @@ side condition that the denominator is non-zero. Run `krun --search` on the
 program `1 / 0 + 2 / 1` and observe all possible outputs of the program. How
 many are there total, and why?
 
-2. Modify your solution from lesson 1.11 problem 2 to remove the `eval`
-function and instead evaluate expressions from left to right using the
+2. Rework the evaluator function from Lesson 1.11, Exercise 2 to evaluate expressions from left to right using the
```

Comment:
I don't think this change is correct. I presume that Lesson 1.11 has a *function* that evaluates expressions, but this exercise is looking to do so with *rewriting* instead.

Reply:
The problem is the sentence is confusing as is. Unless I did something additional by accident, my solution to Lesson 1.11 Problem 2 evaluates expressions both with and without `eval`. But what we can do is to reference the problem that is referenced by Lesson 1.11 Problem 2, where there is no eval and write it straight away like this:

2. Rework the solution to Lesson 1.9, Exercise 2 to evaluate expressions from left to right using the `seqstrict` attribute.

Reply:
@Baltoli Let me know if the current change works, thanks!

Reply:
Yes, as written in the comment that's much clearer!